crates/typed-sled/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "typed-sled"
+version = "0.1.0"
+edition = "2024"
+
+[lints]
+workspace = true
+
+[dependencies]
+dashmap = "6.1.0"
+sled = "0.34.7"
```

Comment:
Can you consolidate this in the root `Cargo.toml` and then update this `Cargo.toml` and the one in `bin/alpen-cli` to use the `workspace` as well: https://github.com/alpenlabs/alpen/blob/fcae36d817f7195ae34ad1e616c507f793e46cdd/bin/alpen-cli/Cargo.toml#L45